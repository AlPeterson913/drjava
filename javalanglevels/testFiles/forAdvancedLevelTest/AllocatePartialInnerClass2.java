// Language Level Converter line number map: dj*->java. Entries: 7
//     1->3         2->4         3->5         4->6         5->7         6->8         7->9    
class AllocatePartialInnerClass2 {
  class MyInnerClass2 {
  }
  
  MyInnerClass2 c = new MyInnerClass2();
  
}
