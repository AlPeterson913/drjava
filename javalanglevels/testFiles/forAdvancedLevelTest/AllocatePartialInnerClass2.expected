class AllocatePartialInnerClass2 { //[1]
  class MyInnerClass2 { //[2]
  } //[3]
  //[4]
  MyInnerClass2 c = new MyInnerClass2(); //[5]
  //[6]
} //[7]